cloud/terraform/terraform_configurator.py

```diff
@@ -115,3 +116,15 @@ def remove_configuration(self):
 
     def set_aws_profile(self, profile):
         self.aws_profile = profile
+
+    def get_username_by_instance_name(self, name):
```

Comment:
What if we search for AMI id? They should be unique and we don't need to format them with a char replacement.

Reply:
I suggest renaming to:
```suggestion
    def get_instance_username_by_ami_id(self, ami_id):
```

Reply:
Please dismiss this comments.